packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts

```diff
@@ -263,6 +263,7 @@ export default VSelect.extend({
         this.selectItem(curItem)
       }
 
+      this.$emit('item:deleted', curItem)
```

Comment:
We're using `click:row` etc in other places, maybe it should be `delete:item` here too?

Reply:
`click:row` is describing an action that's taking place. `item:deleted` is describing an action that has *already* taken place.